packages/components/src/Tooltip/Tooltip.test.tsx

```diff
@@ -3,6 +3,8 @@ import { fireEvent, render, screen, waitFor } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import { Tooltip } from ".";
 
+jest.unmock("@popperjs/core");
```

Comment:
🎉 
Since the library mocks can be a bit hidden, consider adding a comment here to explain why we're unmocking the popper library.